EasyInventory/src/main/resources/static/css/welcome.css

```diff
@@ -4,7 +4,7 @@ body,html {
 }
 
 #header{
-    margin: 0px;
+    margin: 0;
```

Comment:
px is redundantt

Reply:
Is it?
There are several options here (em, px, in, cm).  If nothing else, I'd include px for documentation.
As long as you're here, px is often considered a measurement to not use.   em is preferred, as it is relative.